packages/mds-ingest-service/repository/migrations/1660590738992-DropTelemetryAnnotations.ts

```diff
@@ -0,0 +1,11 @@
+import type { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class DropTelemetryAnnotations1660590738992 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`DROP TABLE "telemetry_annotations"`)
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    throw new Error('Not cannot undor table drop.')
+  }
```

Comment:
Throwing here may cause tests to fail because they run/revert all migrations. In practice we don’t ever revert IRL.

Reply:
@drtyh2o Do you think it'd be safe to have the body of `down` be empty then?

Reply:
Yes. Just put in a comment

Reply:
Thanks! all good suggestions.